### DISCUSSION TITLE
Players join teleporting to island

### DISCUSSION THREAD

**Discussion by AstroZer01** [BG-Software-LLC/SuperiorSkyblock2#2188] [Pending Feature Requests]
Not sure if it's a bug, or just a missing feature.
But whenever a player disconnects, when they join again they are teleported back to there island.
Sometimes the player is in the middle of a dungeon, in a mine or doing anything at all.
They disconect or leave, when they join back they are teleported back the island and lose the progress they where doing.
While searching through the config files i could not find anything to disable the automatic teleport to island on server join.

I don't know if it's a bug, a feature, or something unintended but if there could be atleast a config option to disable it, it would be great.

**Comment by AstroZer01** [comment]
Debuging i discovered that it only happens when using slimeworldmanager module, when not using it players do not login to there island directly

**Comment by OmerBenGera** [comment]
Open an issue under the slime worlds module.
